docs/ibc/light-clients/client-state.md

```diff
@@ -1,3 +1,75 @@
 <!--
 order: 2
--->
\ No newline at end of file
+-->
+
+# Implementing the `ClientState` interface
+
+Learn how to implement the [`Client State`](https://github.com/cosmos/ibc-go/blob/v6.0.0-rc1/modules/core/exported/client.go#L40) interface.
```

Comment:
I know we usually want to link to a specific tag but linking to v6 seems weird to me because its the old interface. I guess we can update all links when there is a v7 tag available. 

```suggestion
Learn how to implement the [`ClientState`](https://github.com/cosmos/ibc-go/blob/v6.0.0-rc1/modules/core/exported/client.go#L40) interface.
```

Reply:
Yeah I think we can update this in the next release.